src/components/core/createPost/index.tsx

```diff
@@ -170,38 +174,36 @@ const CreatePost = () => {
     }
   };
 
+  const handleKeyPress = useCallback((event: KeyboardEvent) => {
+    if (event.ctrlKey && event.key === "Enter") {
```

Comment:
There is a utility which is already define, please use that instead of recreating the same

Reply:
I looked into it - but the helper inctrlKey isn't working with this as the editor is a separate entity . and events aren't working on it . and thats the reason i had it outside the form - becoz upon using the functions its submitting the form_

I'll discuss on this with you on discord